styles/globals.css

```
@@ -55,4 +54,63 @@ body {
 
 .right-navbar-container a {
   @apply text-violet-700 text-xl;
-}
\ No newline at end of file
+}
+
+/* ========== Sidebar Component ========== */
+
+.top-sidebar-container span,
+h3 {
+  @apply text-white;
+}
+
+.top-sidebar-container :first-child {
+  @apply text-2xl font-semibold;
+}
+
+.top-sidebar-container {
+  @apply pt-12 pl-12 flex flex-col gap-y-2;
+}
+
+.side-menu {
+  @apply flex gap-x-3 pt-4 pb-4 pl-6 text-2xl;
+}
+
+.sidemenu-border {
+  @apply pt-6 w-1/2 border-b-2 border-zinc-100;
+}
+
+.side-menu-icon {
+  @apply text-3xl;
```

ลองแก้ config icon ตามนี้ดู มันจะเท่ากันมากกว่า
```suggestion
  @apply h-5 w-5 self-center justify-self-center;
```

---

ลองแก้ config icon ตามนี้ดู มันจะเท่ากันมากกว่า มันจะดูตรงกันกว่า
```suggestion
  @apply h-5 w-5 self-center justify-self-center;
```